src/NServiceBus.Persistence.AzureStorage/Config/TableEntityPartitionKey.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using Persistence.AzureStorage;
+
+    /// <summary>
+    ///
```

Comment:
While the name is self-evident, probably a good idea to let the summary describe what this stands for.